server/guides/other/SETUP.md

```diff
@@ -0,0 +1,232 @@
+
+This changes it to a batch file so we can run the server jar. We are ready to run the server now! Double click the run.bat file and it should come up with a new terminal window and after a while say "Agree to the Minecraft EULA".
+
+![Agree to eula](/images/server/guides/other/SETUP/AGREE_TO_EULA.png)
+
+Close out of the terminal window and find a file called `eula.txt` that was generated. Open it and change `eula=false` to `eula=true`, save it then double click `run.bat` again.
+
+![EULA file](/images/server/guides/other/SETUP/EULA_FILE.png)
+
+It may take a while for all the files to generate so be patient, once it's done your server should be good to go!
+To join the server open Minecraft, go to multiplayer and add a new server, in the server IP add "localhost", click done and join your very own Minecraft server!
+
+![Add localhost](/images/server/guides/other/SETUP/ADD_LOCALHOST.png)
+
+If you would like to be able to access the server from outside your local network, you're going to have to do what's called Port Forwarding, essentially allowing traffic to flow in and out through your router through a specific port to the computer that you're running the server on. Keep in mind this varies from ISP to ISP and from router to router, so a good guide dependent on your router is [here](https://portforward.com/). By default you're going to want to port forward the port 25565 for both the TCP and UDP protocols. To check whether the port forward fully worked, we recommend heading over to this [website](https://www.yougetsignal.com/tools/open-ports/) while your minecraft server is open (preferably after a restart post port forwarding), plug in the port you'd like to check, and if it says open you're good! If not, there might be something you're going to have to redo along the process.
+
+Another reason a port forward might have failed is the windows firewall. To configure this, navigate to the windows defender control panel screen, and head over into advanced settings (you will need admin to do this). Go into inbound rules and outbound rules, and create a new rule for each, name it whatever you like, and make sure that it is allowing the port 25565 on BOTH TCP and UDP (you might need to set up 2 rules for each side). Once that is done, windows should no longer block a server running on that port on your machine.
+
+Note: some ISPs have built in security funcitons in the router itself (looking at you xFinity), and as such they may block people from connecting to the server. While we do not recommend turning off a security function for the everyday user, in some instances it is required.
+
+This process is slightly different for a modded server so we will go over it here!
+
+# Setup For Linux
+
+> Warning: You should prefer using your package manager (for example `apt` on Ubuntu and Debian) to install java because it's more organized with the package manager
+
+You can do this using the following command
+
+--> `sudo apt install openjdk-xx-jre`
```

Comment:
Maybe add either commands for other distros/ package managers or a note that not every distro uses `apt`